pkg/commands/options/build.go

```diff
@@ -24,7 +26,12 @@ import (
 type BuildOptions struct {
 	// BaseImage enables setting the default base image programmatically.
 	// If non-empty, this takes precedence over the value in `.ko.yaml`.
-	BaseImage            string
+	BaseImage string
+
+	// Dir allows for setting the working directory for invocations of the `go` tool.
+	// Empty string means the current working directory.
+	Dir string
```

Comment:
Can we rename this to `WorkingDir` or `WorkingDirectory`? I think that would help with the name collision for https://github.com/google/ko/pull/365

Reply:
Done. Also squashed the commits.